_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why it is so important to do analysis pre-gsoc and
```

Comment:
Just a quick question, should `gsoc` be capital?
I am not sure.

Reply:
Yep, that has to be in capital.
My bad, I will correct that by changing it to the `GSoC`.